### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Kilian Froidevaux

### DISCUSSION THREAD

**Discussion by kilianfroideva** [heig-vd-dai-course/heig-vd-dai-course#253] [Show and tell]
https://github.com/kilianfroideva/AboutMe.git

**Comment by ludelafo** [comment]
Nice profile! However, if you want to be visible when someone accesses your profile (https://github.com/kilianfroideva), you must rename your repository to `kilianfroideva`.

What kind of games are you developing with Godot? :)
